app/views/cart_items/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1> Cart Contents: </h1>
+<table class="bordered striped highlight centered responsive-table">
+  <thead>
+    <tr>
+      <th data-field="Image">Image</th>
+      <th data-field="Title">Title</th>
+      <th data-field="Description">Description</th>
+      <th data-field="Travesty">Travesty</th>
+      <th data-field="Price">Price ($)</th>
```

Comment:
If number_to_currency method is used in the partial for each cart item, it adds a $ sign for the prices and it isnt needed on the table headers.